src/App.tsx

```diff
@@ -20,17 +33,21 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                // value={query}
```

Comment:
Don't comment `value` property
It's important to match the state, that you changing  in `onChange`